jc/parsers/iftop.py

```diff
@@ -0,0 +1,622 @@
+"""jc - JSON Convert `iftop` command output parser
+
+No `iftop` options are supported.
```

Comment:
Are there only certain options that are supported or can we remove this if all options are supported? From the example and tests it looks like at least some options are supported.